src/deploymentmanager.ts

```diff
@@ -377,6 +377,9 @@ export class DeploymentManager implements IDeploymentManager {
         if (this._parameters.adminPassword) {
             this._parameters.adminPassword.value = answers.adminPassword;
         }
+        if (this._parameters.subscriptionId) {
```

Comment:
Why we need this template parameter? You can get it by call `subscription().subscriptionId` from ARM template.

Reply:
updated